### DISCUSSION TITLE
Future of DWMBlurGlass?

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#224] [Q&A] [answered]
Hi, I completely understand if this is an annoying, overasked question and don't want to be bothered about what you're doing or feeling pressured to give an update, in which case you can close this thread and ignore it. I'm just curious are there future update plans to DWMBlurGlass, or is it finished? Do we just need to create and share our own forks, like some have been doing, if we want extra features or changes? Not asking for ETA; just wondering what the future is for it?

**Comment by ALTaleX531** [comment, marked as answer]
https://github.com/Maplespe/DWMBlurGlass/pull/222#issuecomment-2017962352

**Reply by TheInsane101** [reply]
Thanks!
